bots/elasticsearch/tasks.py

```diff
@@ -18,6 +21,22 @@ def safe_substr(value, length=32000):
         return str(value)[:length]
     return None
 
+def add_suggest(obj):
+    input = obj['name']
+    if input:
+        # Elastic's completion suggester only looks at prefixes. If we give each
```

Comment:
Any reason to not just use a combination of `preserve_position_increments` and `preserve_separators` set to `False`?
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters-completion.html#completion-suggester-mapping